src/components/card/Description.vue

```diff
@@ -252,6 +252,7 @@ export default {
 		flex-shrink: 1;
 		overflow: scroll;
 		max-height: 50vh;
+		min-height: 50vh;
```

Comment:
I think it might even be better if we just remove any height specification and also the overflow scroll, as currently we have two scrollbars with that:

Removing the overflow also seems to fix the tooltip flickering:
![Uploading Screenshot 2022-02-28 at 12.49.44.png…]()